x-pack/plugins/fleet/server/services/epm/packages/install.ts

```diff
@@ -383,7 +377,8 @@ export async function createInstallation(options: {
     },
     { id: pkgName, overwrite: true }
   );
-  return [...installedKibana, ...installedEs];
+
+  return created;
```

Comment:
If I understand the existing code correctly (both reading & running with logging), we always passed empty arrays for the two `installed*` params. Which makes this `[ ...[], ...[] ]` or `[]`

Even if they had values, rather than return the caller data they gave, return the result of the function.

The return value isn't used by the one caller so this is safe and unnoticed.